.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      if: matrix.python-version == '2.7'
+      run: |
+        python -m pip install --upgrade pip
+        pip install mock
```

Comment:
Remove "mock" from `tests_require` in setup.py, I think that's what trips some of the tests up.